src/AudioPlayer.cpp

```diff
@@ -399,14 +404,19 @@ void AudioPlayer_Task(void *parameter) {
 			}
 		}
 
-		trackQStatus = xQueueReceive(gTrackQueue, &gPlayProperties.playlist, 0);
+		Playlist *newPlaylist;
+		trackQStatus = xQueueReceive(gTrackQueue, &newPlaylist, 0);
 		if (trackQStatus == pdPASS || gPlayProperties.trackFinished || trackCommand != NO_ACTION) {
 			if (trackQStatus == pdPASS) {
 				if (gPlayProperties.pausePlay) {
 					gPlayProperties.pausePlay = false;
 				}
 				audio->stopSong();
-				Log_Printf(LOGLEVEL_NOTICE, newPlaylistReceived, gPlayProperties.numberOfTracks);
+
+				// destory the old playlist and assign the new
```

Comment:
spelling "destroy" ;-)

Reply:
what, does one not write destory when they want to say destory? 😄